ln/subscribe_invoice.ts

```diff
@@ -47,18 +51,20 @@ const subscribeInvoice = async (bot, id, resub) => {
             rate
           );
         }
-        order.invoice_held_at = Date.now();
+        order.invoice_held_at = new Date();
         order.save();
       }
       if (invoice.is_confirmed) {
         const order = await Order.findOne({ hash: id });
+        if (order === null) return;
         logger.info(
           `Order ${order._id} - Invoice with hash: ${id} was settled!`
         );
         order.status = 'PAID_HOLD_INVOICE';
         await order.save();
         const buyerUser = await User.findOne({ _id: order.buyer_id });
         const sellerUser = await User.findOne({ _id: order.seller_id });
+        if (buyerUser === null || sellerUser === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program